doc/developer/Makefile

```diff
@@ -0,0 +1,219 @@
+# Makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
```

Comment:
This should first check if the environment variable is set and only use these default values when not